_includes/footer-content.html

```diff
@@ -4,9 +4,9 @@
       <p>About</p>
       <ul class="list-unstyled">
         <li><a href="https://github.com/bazelbuild/bazel/wiki/Bazel-Users">Who's using Bazel</a></li>
-        <li><a href="https://bazel.build/roadmap.html">Roadmap</a></li>
-        <li><a href="https://bazel.build/contributing.html">Contribute</a></li>
-        <li><a href="https://bazel.build/governance.html">Governance Plan</a></li>
+        <li><a href="/roadmap.html">Roadmap</a></li>
```

Comment:
I prefer using absolute paths here, so that we can exactly copy paste this file without modifications between our 3 repositories.
If available, you could use a variable ({{site_url}}?) instead of `https://bazel.build`